VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-global-styles.scss

```diff
@@ -142,10 +142,20 @@
   color: white;
 }
 
+.video-wrapper, .panel-wrapper {
+  height: calc(100% - 4.094375rem);
+}
```

Comment:
Unfortunately without some additional refactoring this has to be hard coded for now.